src/lib/protocols/connections/ConnectionService.test.ts

```diff
@@ -275,17 +275,20 @@ describe('ConnectionService', () => {
       ConnectionState.Responded,
       ConnectionState.Complete,
     ];
-    test.each(invalidConnectionStates)('throws an error when connection state is %s and not INVITED', state => {
-      expect.assertions(1);
+    test.each(invalidConnectionStates)(
+      `throws an error when connection state is %s and not ${ConnectionState.Invited}`,
```

Comment:
I'm still not sure about using constants in the test message. I like it because it does reuse the enum value, on the other side if you have that value incorrect, the test doesn't reveal that.

For example, if I have `ConnectionState.Invited = 'requested'` then it could be confusing

Reply:
I agree! I'll make an additional PR to fix that

Reply:
After thinking about this using the string value everywhere also doesn't seem right. I'll make a test per state that tests the string value. Then we can use the enum value everywhere else.